core/tauri-utils/src/config.rs

```diff
@@ -1248,6 +1248,7 @@ pub struct RemoteDomainAccessScope {
   /// The list of window labels this scope applies to.
   pub windows: Vec<String>,
   /// The list of plugins that are allowed in this scope.
+  /// The names should be without the `tauri-plugin-` prefix, for example `"store"` for `tauri-plugin-store`.
```

Comment:
This still is confusing because a plugin could have the name `tauri-plugin-foo` but in code it is actually, `tauri::plugin::Builder::new("bar").build()`

Reply:
Yeah, i thought about that too, but no idea at all how to make this easy to grasp for non-plugin authors & non-rust users so i went with the hotfix like wording for now :/

Reply:
yeah...